stable/dask/templates/dask-jupyter-deployment.yaml

```
@@ -26,14 +26,16 @@ spec:
         release: {{ .Release.Name | quote }}
         component: jupyter
     spec:
+      imagePullSecrets:
+{{ toYaml .Values.jupyter.image.pullSecrets | trimSuffix "\n" | indent 8 }}
```

I'd suggest you go with this:

```suggestion
        {{- toYaml .Values.jupyter.image.pullSecrets | nindent 8 }}
```

Please apply throughout the chart.

---

Changes applied, thanks for the suggestion!